webapp/src/Entity/Team.php

```diff
@@ -58,6 +57,13 @@ class Team extends BaseApiEntity implements ExternalRelationshipEntityInterface
      */
     private $name;
 
+    /**
+     * @var string|null
+     * @ORM\Column(type="string", name="display_name", length=255, options={"comment"="Team display name", "collation"="utf8mb4_bin"},
+     *                            nullable=true)
```

Comment:
does the indentation only look weird in the github UI?

Reply:
What is 'weird'? The nullable is now with name=, but probably should be with type=?

Reply:
Yes, exactly.